client/extensions/wp-super-cache/style.scss

```diff
@@ -151,3 +151,23 @@
 .wp-super-cache__cdn-fieldsets {
 	margin-left: 40px;
 }
+
+.wp-super-cache__main {
```

Comment:
I added this rule like this, because I needed to override the specificity of this rule:

```
.wp-super-cache__main .button {
    margin-right: 8px;
}
```